### PR TITLE
Fix test set for Japanese Vowels

### PR DESCRIPTION
**Pull request by off99555**
#### Reference Issues/PRs
#1171 

#### What does this implement/fix? Explain your changes.
Fix test set, previously test set was the same as training set.

#### Does your contribution introduce a new dependency? If yes, which one?
No

#### What should a reviewer concentrate their feedback on?
Check that the test set is now different than training set.

### CODE REVIEW

**Comment by TonyBagnall** [comment]
Japanese vowels is used in the tests so we will need to update them. @MatthewMiddlehurst I think its your tests failing, could you update them for the correct Japanese vowel data?

**Comment by MatthewMiddlehurst** [comment]
I'm pretty sure `test_muse.py` is the only test using the dataset. Seems to be crashing now that the datasets are different, not an issue with changing accuracy.

On my branch changing the dataset to BasicMotions also causes it to crash.

`dictionary_based_classification.ipynb` also uses it, so may want to rerun that as well.

**Comment by TonyBagnall** [comment]
hi, so to fix this I suggest either work out the correct output for test_muse.py and update that file in this PR, or if you prefer remove test_muse.py on this PR and we can put it back for you once your PR is in.

**Comment by MatthewMiddlehurst** [comment]
Alternatively change the dataset to `load_basic_motions`, you will have to lower the indices size to i.e. `indices = np.random.RandomState(0).permutation(20)` though.

JapaneseVowels is an unequal length dataset according to the test set (but not the train, the headers now mismatch a bit fyi), can WEASEL/MUSE handle that?

**Comment by TonyBagnall** [comment]
That's a good idea, also need to flag this issue of tags on the data repo

**Comment by MatthewMiddlehurst** [comment]
The train set on the website has a similar header to the test set from this PR, the current train set in sktime is a bit off though. Not really sure why they are different.

**Comment by MatthewMiddlehurst** [comment]
current sktime header:
header1

new header:
header2

ok yeah this is causing an issue

**Comment by fkiraly** [comment]
It seemed like this was close to complete, except some minor code formatting issues - are you still working on this @off99555?

**Comment by off99555** [comment]
@fkiraly No, someone please commit to this PR. I only thought that just uploading the dataset is enough. If I have to modify anything in the file, probably someone else who understand the process should do it.

**Comment by fkiraly** [comment]
@MatthewMiddlehurst, do you want to take this over then? I think the file has been fixed?

**Comment by MatthewMiddlehurst** [comment]
@fkiraly Me or Tony will sort this at some point, turns out theres a bit more than just replacing the duplicate file. Files from the TSC website and those in the repo are a bit different and the swap causes a test to break.

**Comment by MatthewMiddlehurst** [comment]
Both dataset files have been replaced with the unequal length versions from timeseriesclassification.com in #1278.
